validator/app_versions.json

```diff
@@ -63,7 +63,10 @@
          "41.0a1", "41.0a2", "41.0", "41.*",
          "42.0a1", "42.0a2", "42.0", "42.*",
          "43.0a1", "43.0a2", "43.0", "43.*",
-         "44.0a1", "44.0a2", "44.0", "44.*"]
+         "44.0a1", "44.0a2", "44.0", "44.*",
+         "45.0a1", "45.0a2", "45.0", "45.*",
+         "46.0a1", "46.0a2", "46.0", "46.*"]
```

Comment:
Ah yeah, I wanted to do them into one PR first so I put this part here and have the same change in both PRs.